web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java

```diff
@@ -793,6 +902,22 @@ private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID
         fail("No CSV record with PID " + expectedPid.getId() + " present");
     }
 
+    private void assertCsvContentIsCorrect(List<CSVRecord> csvList, PID folderPid, Map<String, PID> pidList) {
```

Comment:
pidList always contains exactly 2 pids with the same name, so I'd say we should just pass separate variables in for workPid and filePid instead, which would be consistent with folderPid.